study.md

```diff
@@ -135,13 +134,25 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.select { |o| o > 1 }
+
+the below code is for me for a later reference:
+def create_1_to_10_odd_array
+    array = [1]
+    i = 3
+    while i < 10
+        array.push i
+        i += 2
+    end
+
+    array
+end
 ```
 
 JavaScript:
 
 ```javascript
 const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
-// your answer here
+array.filter((x) => x % 2 != 0)
```

Comment:
This works in this example, but remember to use the strict inequality operator (`===`) instead.